ci/buildspec.yml

```diff
@@ -0,0 +1,30 @@
+# Build Spec for AWS CodeBuild CI
+
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - apt-get update
+      - apt-get install -y curl
+      - apt-get install -y libgfortran3
+      - pip install pip -U
+      - pip install flask
+      #- pip install -U mxnet
+      - pip install Pillow
+      - pip install requests
+      - pip install flask-cors
+      - pip install fasteners
+      - pip install jsonschema
+      # - pip install onnx -U
+      # - pip install onnx-mxnet -U
+      - pip install -U -e .
+  build:
+    commands:
+      - python -m pytest mms/tests/unit_tests
```

Comment:
Are the integration tests taken out intentionally?
Can you please add a comment with a link to an open issue so this does not get missed?

Reply:
done